src/style.css

```diff
@@ -2,0 +1,40 @@
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-image: url(./images/star_desactived.png);
+  background-position: center;
+  background-repeat: no-repeat;
+}
+
+.stars--0 .stars__star:nth-child(-n + 0) {
```

Comment:
It seems there is a mistake with the selector '.stars--0 .stars__star:nth-child(-n + 0)'. The ':nth-child(-n + 0)' part will never select any elements because it's asking for the 0th child and there is no such thing. If you want to select no stars in a 'stars--0' case, you can omit the rule entirely.